18_regexp_lab/01.py

```diff
@@ -1,12 +1,35 @@
-"""
-Write a program that reads data
-from property files.
-Each line in the file can either be:
-    An empty line
-    A comment line (Start with #)
-    A property line (of the form key = value)
-
-Write a program that takes a property file name and key
-as command line arguments and prints the requested value
-"""
-
+#!/usr/bin/env python
+
+__author__ = 'Eabel'
+
+import sys
+import os
+import argparse
+import re
+
+
+########################################################################################################################
+#
+#
+def main():
+    parser = argparse.ArgumentParser(description='This program receive config file path and varible name and prints the varible value')
+    
+    parser.add_argument("config_path", help="path to config file")
+    parser.add_argument("varible_name", help="varible name")
+    args = parser.parse_args()
+
+    try:
+        regex_str = '\s*' + args.varible_name + '\s*=\s*(.*)'

```

Comment:
I was referring to re.split (to handle whitespace):
https://docs.python.org/2/library/re.html#re.split

Also you need a regex to ignore comments